src/include/rados/librados.h

```diff
@@ -3124,7 +3124,28 @@ CEPH_RADOS_API void rados_read_op_omap_get_vals(rados_read_op_t read_op,
 				                const char *filter_prefix,
 				                uint64_t max_return,
 				                rados_omap_iter_t *iter,
-				                int *prval);
+				                int *prval)
+  __attribute__((deprecated)); /* use v2 below */
```

Comment:
These _read_op_* methods are used to build up a multi-op transaction.. we can't do magic client-side looping for them.